### DISCUSSION TITLE
Sync app outdated

### DISCUSSION THREAD

**Discussion by DeeKahy** [TechbeeAT/jtxBoard#589] [General]
I cannot sync with my server because I need DAVx5 4.3.3 or later, but that DAV version isn't available on f-droid or the play store. I am using android 13.

**Comment by patrickunterwegs** [comment]
Hi @DeeKahy, are you using jtx Board from the Beta-Channel of Google Play? You should also join the beta-channel then of DAVx5 to get 4.3.3!

**Reply by DeeKahy** [reply]
Oh yeah, now it works, thank you! I forgot that i switched to the beta channel :/
